go.mod

```diff
@@ -11,21 +11,21 @@ replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.5
 require (
 	github.com/golang/sync v0.0.0-20180314180146-1d60e4601c6f
 	github.com/pmezard/go-difflib v1.0.0
-	istio.io/istio v0.0.0-20220819144046-eda724aef4c7
-	istio.io/pkg v0.0.0-20220815202617-8f42645c6e49
-	k8s.io/apimachinery v0.24.2
-	k8s.io/client-go v0.24.2
+	istio.io/istio v0.0.0-20220826204218-8f0fc3e2d673
```

Comment:
There are a number of extra changes in the go.* files. https://github.com/istio/istio.io/pull/11754 has a smaller set of changes. It looks like these changes match the main istio/istio branch while we are testing against the 1.15 branch. If you would remove the extra changes.

Reply:
The additional changes will come in as part of the istio/istio.io branch cut to release-1.15 when 1.15.0 is released.

Reply:
remove extra changes base on #11754 .